planner/core/encode.go

```diff
@@ -40,6 +141,7 @@ type planEncoder struct {
 }
 
 // EncodePlan is used to encodePlan the plan to the plan tree with compressing.
+// Deprecated: FlattenPhysicalPlan() + EncodeFlatPlan() is preferred.
 func EncodePlan(p Plan) string {
```

Comment:
Now it's only used in two benchmark tests and one true unit test, we can remove it actually?

Reply:
Yes.
I'll make sure all those methods can be removed safely and remove them in future PRs.